meseq.py

```diff
@@ -343,14 +349,23 @@ def draw(self):
         y = 0
         for row in self.matrix.rows:
             y += STEP
-            for i in range(len(row)):
+            for i, node in enumerate(row):
+                if node is None:
+                    continue
+
+                if node.type in [ NT_LIFELINE, NT_MSG_RECV, NT_MSG_SEND, NT_MSG_LOST, NT_CREATE, NT_BIDIRECTIONAL ]:
+                    self.lifeLine(x, y-STEP/2, y+STEP/2)
+                elif node.type == NT_TERMINATE:
+                    self.lifeLine(x, y-STEP/2, y)
                 x = 2*STEP + i * STEP * 3
-                node = row[i]
 
+            for i, node in enumerate(row):
```

Comment:
Please do not walk twice through the row.
A drawback of the 2 'for' loops is some duplicated code : 
    x = 2*STEP + i * STEP * 3